service-tests/vscode.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'vscode', title: 'VS Code Marketplace' });
+module.exports = t;
+
+t.create('installs')
+.get('/installs/ritwickdey.LiveServer.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('Installs', 'installs'),
+  value: Joi.string().regex(/^[0-9]+[K|M]?$/i)
+}));
+
+t.create('rating')
+.get('/rating/ritwickdey.LiveServer.json')
+.expectJSONTypes(Joi.object().keys({
+  name: Joi.equal('rating', 'Rating'),
+  value: Joi.string()
```

Comment:
This is very broad, and matches e.g. the string `'undefined'`. What are the expected values here?

Reply:
Ok! I'll add validation